lib/geared_pagination/page.rb

```diff
@@ -35,7 +35,11 @@ def only?
     end
 
     def last?
-      number >= recordset.page_count
+      number == recordset.page_count
+    end
```

Comment:
> last? now returns true for exactly the last page, and only for that. Pages that don't "exist", because they have numbers that are greater than the total page count, will have last? return false.

What made this change necessary?

Reply:
It was HEY's implementation of infinite pagination, but I haven't found out why. I plan to change HEY to use `before_last?` instead.